client/my-sites/domains/domain-management/edit/domain-types/transfer-in-domain-type.jsx

```diff
@@ -0,0 +1,231 @@
+import { domainManagementTransferInPrecheck } from 'my-sites/domains/paths';
+import { isCancelable } from 'lib/purchases';
+import { cancelPurchase } from 'me/purchases/paths';
+import RemovePurchase from 'me/purchases/remove-purchase';
+import { INCOMING_DOMAIN_TRANSFER_STATUSES } from 'lib/url/support';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class TransferInDomainType extends React.Component {
+	getVerticalNavigation() {
+		return <VerticalNav>{ this.removeTransferNavItem() }</VerticalNav>;
+	}
+
+	removeTransferNavItem() {
+		const { domain, isLoadingPurchase, purchase, selectedSite, translate } = this.props;
+
+		if ( ! domain.currentUserCanManage ) {
+			return null;
+		}
+
+		if (
+			domain.transferStatus === transferStatus.PENDING_OWNER ||
+			domain.transferStatus === transferStatus.PENDING_REGISTRY
+		) {
+			return null;
+		}
+
+		let title;
+
+		if ( domain.transferStatus === transferStatus.CANCELLED ) {
+			title = translate( 'Remove transfer from your account' );
+		} else {
+			title = translate( 'Cancel transfer' );
+		}
+
+		if ( isLoadingPurchase ) {
+			return <VerticalNavItem isPlaceholder />;
+		}
+
+		if ( ! selectedSite || ! purchase ) {
+			return null;
+		}
+
+		if ( isCancelable( purchase ) ) {
+			const link = cancelPurchase( selectedSite.slug, purchase.id );
+
+			return <VerticalNavItem path={ link }>{ title }</VerticalNavItem>;
+		}
+
+		return (
+			<RemovePurchase
+				hasLoadedSites={ true }
+				hasLoadedUserPurchasesFromServer={ true }
+				site={ selectedSite }
+				purchase={ purchase }
+				title={ title }
+				hideTrashIcon={ true }
+				displayButtonAsLink={ true }
+			/>
+		);
+	}
+
+	resolveStatus() {
+		const { domain, translate } = this.props;
+		const { transferStatus: status } = domain;
+
+		if ( status === transferStatus.PENDING_START ) {
+			return {
+				statusText: translate( 'Action required' ),
+				statusClass: 'status-error',
+				icon: 'info',
+			};
+		} else if ( status === transferStatus.CANCELLED ) {
+			return {
+				statusText: translate( 'Transfer failed' ),
+				statusClass: 'status-error',
+				icon: 'info',
+			};
+		}
+
+		return {
+			statusText: translate( 'Transfer in progress' ),
+			statusClass: 'status-success',
+			icon: 'cached',
+		};
+	}
+
+	startTransfer = () => {
+		const { domain, selectedSite } = this.props;
+		page( domainManagementTransferInPrecheck( selectedSite.slug, domain.name ) );
+	};
+
+	renderPendingStart() {
+		const { domain, translate } = this.props;
+
+		return (
+			<>
+				<p>
+					{ translate(
+						'We need you to complete a couple of steps before we can transfer %(domain)s from your ' +
+							'current domain provider to WordPress.com. Your domain will stay at your current provider ' +
+							'until the transfer is completed.',
+						{
+							args: {
+								domain: domain.name,
+							},
+						}
+					) }
+				</p>
+
+				<Button primary onClick={ this.startTransfer }>
+					{ translate( 'Start transfer' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Start Transfer' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/domain-status-inbound-transfer)